Etabs_Adapter/CRUD/Read/Results/GlobalResults.cs

```diff
@@ -56,6 +56,9 @@ public IEnumerable<IResult> ReadResults(GlobalResultRequest request, ActionConfi
                 case GlobalResultType.Reactions:
                     return GetGlobalReactions();
                 case GlobalResultType.ModalDynamics:
+
+                case GlobalResultType.StoryDrifts:
```

Comment:
```suggestion
                case GlobalResultType.StoreyDrifts:
```

Reply:
@peterjamesnugent 
change incorporated in the code.